src/Sulu/Bundle/TestBundle/Resources/dist/orm_mysql.yml

```diff
@@ -7,3 +7,29 @@ doctrine:
         user:     "%database_mysql_user%"
         password: "%database_mysql_password%"
         charset:  UTF8
+    orm:
+        dql:
+            datetime_functions:
+                date: Oro\ORM\Query\AST\Functions\SimpleFunction
+                time: Oro\ORM\Query\AST\Functions\SimpleFunction
+                timestamp: Oro\ORM\Query\AST\Functions\SimpleFunction
+                convert_tz: Oro\ORM\Query\AST\Functions\DateTime\ConvertTz
+            numeric_functions:
+                timestampdiff: Oro\ORM\Query\AST\Functions\Numeric\TimestampDiff
+                dayofyear: Oro\ORM\Query\AST\Functions\SimpleFunction
+                dayofmonth: Oro\ORM\Query\AST\Functions\SimpleFunction
+                dayofweek: Oro\ORM\Query\AST\Functions\SimpleFunction
+                week: Oro\ORM\Query\AST\Functions\SimpleFunction
+                day: Oro\ORM\Query\AST\Functions\SimpleFunction
+                hour: Oro\ORM\Query\AST\Functions\SimpleFunction
+                minute: Oro\ORM\Query\AST\Functions\SimpleFunction
+                month: Oro\ORM\Query\AST\Functions\SimpleFunction
+                quarter: Oro\ORM\Query\AST\Functions\SimpleFunction
+                second: Oro\ORM\Query\AST\Functions\SimpleFunction
+                year: Oro\ORM\Query\AST\Functions\SimpleFunction
+                sign: Oro\ORM\Query\AST\Functions\Numeric\Sign
+                pow: Oro\ORM\Query\AST\Functions\Numeric\Pow
+            string_functions:
+                group_concat: Oro\ORM\Query\AST\Functions\String\GroupConcat
+                concat_ws: Oro\ORM\Query\AST\Functions\String\ConcatWs
+                cast: Oro\ORM\Query\AST\Functions\Cast
```

Comment:
hmm i thought about adding all of that because we already use the extension but i could remove all except the grout_concat